### PR TITLE
Adding error states for transfer, account page

### PR DESCRIPTION
**Pull request by hariria**
## Motivation

https://user-images.githubusercontent.com/31301117/166158954-4a513395-ce0e-4e60-840d-1e34ea36f79b.mov

Addressing issues tasks in here: https://github.com/aptos-labs/aptos-core/issues/703

Completed MVP requirements for web-wallet:

- [x] invalid address for transfer behavior @hariria
- [x]  Invalid amount for transfer behavior (ie. what if they send -1 tokens?) @hariria
- [x]  Create new wallet @hariria
- [x]  Sign out / wipe local storage @hariria
- [x]  Access public key / private key @hariria
- [x]  Explore link for resources @hariria 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?
Yes

## Test Plan
See above

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site, and link to your PR here.)

### CODE REVIEW

**Comment by davidiw** [comment]
This is great! Now that we have the updated token standard, it might be worth exploring that and/or dapp concept.

**Review by kent-white — APPROVED** [review]
nits, and random thoughts

File: ecosystem/web-wallet/src/utils/account.ts [hunk 1]
```diff
@@ -21,8 +21,8 @@ export function loginAccount (key: string): Result<AptosAccount, Error> {
   }
 }
 
-export function createNewAccount (): Result<AptosAccount, Error> {
+export function createNewAccount (): AptosAccount {
   const account = new AptosAccount()
   // todo: make request to create account on chain
-  return ok(account)
```
Comment (kent-white) [inline ecosystem/web-wallet/src/utils/account.ts:27]: Also just delete the Result type lol

File: ecosystem/web-wallet/src/components/HelpHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <ChakraLink to="/">
+          <IconButton size="xs" borderRadius="full" aria-label="go back" icon={<ChevronLeftIcon fontSize="md" />} />
+        </ChakraLink>
+        <Text
+          fontSize="md"
+          textAlign="center"
+        >
+          Help
+        </Text>
+        <></>
```
[the remark this answers is not in the document]
Comment (hariria) [reply ecosystem/web-wallet/src/components/HelpHeader.tsx:39]: haha hmmm not sure :D

File: ecosystem/web-wallet/src/pages/Login.tsx [hunk 3]
```diff
@@ -113,15 +110,21 @@ function Login () {
...
-      <Button size="xs" as="a" href='/help' leftIcon={<QuestionIcon />} variant="link">
-        Help
-      </Button>
+      {/* TODO: Fill this in later */}
+      {/* <HStack spacing={2} color={secondaryTextColor[colorMode]}>
+        <QuestionIcon />
+        <ChakraLink to="/help" fontSize="xs">
+          Help
+        </ChakraLink>
+      </HStack> */}
```
[the remark this answers is not in the document]
Comment (hariria) [reply ecosystem/web-wallet/src/pages/Login.tsx:127]: :( @kent-white

**Pull request merged**